internal/embedding/celestia-app_Darwin_arm64.tar.gz

```diff
@@ -0,0 +1 @@
+placeholder
```

Comment:
TODO: leaving like this until we decide exactly how to handle embedding binaries in celestia-app repo.